include/atomics/subnet_cadmium.hpp

```diff
@@ -27,87 +27,96 @@
+	//This is a meta-model, 
+	//it should be overloaded for declaring the "id" parameter
     template<typename TIME>
     class Subnet{
-        using defs=Subnet_defs; // putting definitions in context
+        using defs = subnet_defs; // putting definitions in context
         public:
             //Parameters to be overwriten when instantiating the atomic model
            
             // default constructor
             Subnet() noexcept{
-              state.transmiting     = false;
-              state.index           = 0;
+				state.transmiting = false;
+				state.index = 0;
             }
             
             // state definition
             struct state_type{
-              bool transmiting;
-              int packet;
-              int index;
+				bool transmiting;
+				int packet;
+				int index;
             }; 
             state_type state;
+
             // ports definition
             using input_ports=std::tuple<typename defs::in>;
             using output_ports=std::tuple<typename defs::out>;
 
             // internal transition
             void internal_transition() {
-                state.transmiting = false;  
+				state.transmiting = false;  
             }
 
             // external transition
-            void external_transition(TIME e, typename make_message_bags<input_ports>::type mbs) { 
+            void external_transition(TIME e,
+							         typename make_message_bags<input_ports>::type mbs) { 
                 state.index ++;
-                if(get_messages<typename defs::in>(mbs).size()>1) assert(false && "One message at a time");                
+                if(get_messages<typename defs::in>(mbs).size()>1) {
+					assert(false && "One message at a time");
+				}	
                 for (const auto &x : get_messages<typename defs::in>(mbs)) {
-                  state.packet = static_cast < int > (x.value);
-                  state.transmiting = true; 
+					state.packet = static_cast < int > (x.value);
+					state.transmiting = true; 
                 }               
             }
 
             // confluence transition
-            void confluence_transition(TIME e, typename make_message_bags<input_ports>::type mbs) {
+            void confluence_transition(TIME e,
+				                      typename make_message_bags<input_ports>::type mbs) {
                 internal_transition();
                 external_transition(TIME(), std::move(mbs));
             }
 
             // output function
             typename make_message_bags<output_ports>::type output() const {
-              typename make_message_bags<output_ports>::type bags;
-              Message_t out;
-              if ((double)rand() / (double) RAND_MAX  < 0.95){
-                out.value = state.packet;
-                get_messages<typename defs::out>(bags).push_back(out);
-              }
-              return bags;
+						typename make_message_bags<output_ports>::type bags;
```

Comment:
typename indented to much

Reply:
download the files and open the file in your local editor. you can observe them as aligned with coding conventions.

Reply:
the same thing applies to all the comments. open them in local editor then you can observe them with coding standards

Reply:
I opened your code using sublime,notepad and VS 2015, the issues raised still stand.